.hashibot.hcl

```diff
@@ -11,6 +11,20 @@ poll "closed_issue_locker" "locker" {
   EOF
 }
 
+poll "stale_issue_closer" "closer" {
+    schedule = "0 50 12 * * *"
```

Comment:
this is a brittle aspect of the system 🙈, I'd like to avoid polls all waking at this same time, could we just pick some arbitrary point in the day, say 3:10? `0 10 3 * * *`

Reply:
Sounds fine, updated